template/tsconfig/cypress-ct/tsconfig.app.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "@vue/tsconfig/tsconfig.web.json",
+  "include": ["env.d.ts", "src/**/*", "src/**/*.vue"],
```

Comment:
Maybe we need't "src/**/*.vue"

Reply:
I'm not sure about this. Even though `vue-tsc` doesn't need the `.vue` extension, many TypeScript-related tools are already assumed that only a particular group of "default extensions" are allowed.

For example, I just checked `tsconfck`, it has this assumption too:
https://github.com/dominikg/tsconfck/blob/a46f60c03c088dc2c6a1283b421919a398df6a05/src/util.ts#L174-L177

Vite uses `tsconfck` to get the corresponding typescript configuration for a given file path, because some of the tsconfig options will affect the compilation result (e.g. [preserveValueImports](https://www.typescriptlang.org/tsconfig/#preserveValueImports)).

So removing `.vue` here is a breaking change to those who relied on this behavior in `.vue` files.

---

Let's keep it as-is. After all, it's already a well-established convention in the TypeScript ecosystem.